data/faq.html

```diff
@@ -1062,6 +1272,29 @@ <h3 lang="es">Búsqueda</h3>
 <p></p>
 Nota: solamente puedes encontrar los sitios que han sido mapeados en 
 <a href="http://www.openstreetmap.org">OpenStreetMap</a>.</dd>
+<dd lang="fr">Cliquer sur l'icone de recherche dans l'écran principal. Vous pouvez rechercher en
+utilisant l'un des cas suivants :
+<p></p>
+<b><em>Adresse</em></b> : entrez le N° de la maison, le nom de la rue. Vous pouvez également inquer
+le nom de la ville (si vous ne trouvez toujours pas) et le code postal.
+Par exemple: 41, Rue des Alouettes 75000 Paris.
```

Comment:
This might be the correct postal code but the app still cannot find it (there is no information in osm data and we currently do not throw away parts of a search query).
Can you please try to look for a full address for which the search will return something?